src/strategies/math/index.ts

```diff
@@ -170,7 +169,7 @@ async function resolveOperand(
         strategyOperand.strategy.name
       ].strategy(
         space,
-        network,
+        strategyOperand.strategy.network,
```

Comment:
Hi @RatanRSur Can make it optional? like 
```js
strategyOperand.strategy.network || network
```

Reply:
Good idea. I hope it's ok that I used `??` instead to be more precise